visualization/motion_planning_tasks/src/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ target_include_directories(${MOVEIT_LIB_NAME}
 	PUBLIC $<TARGET_PROPERTY:motion_planning_tasks_properties,INTERFACE_INCLUDE_DIRECTORIES>
 	PUBLIC $<TARGET_PROPERTY:moveit_task_visualization_tools,INTERFACE_INCLUDE_DIRECTORIES>
 	PUBLIC ${catkin_INCLUDE_DIRS}
+	PUBLIC SYSTEM ${OGRE_INCLUDE_DIR}
```

Comment:
This syntax is not possible. You need to add another `target_include_directories` command with the `SYSTEM` keyword present directly after the target name. I ran into the same issue...

Reply:
I thought I tested this locally, but apparently it doesn't even work in cmake 3.19 over here...
Fixing it.